Cargo.toml

```diff
@@ -1,4 +1,101 @@
+[package]
+name = "bob_management"
+version = "0.1.0"
+default-run = "bob_management"
+edition = "2021"
+description = "Bob Management GUI"
+authors = ["Romanov Simeon ArchArcheoss@proton.me", "Artemev Ilia", "Riadinskii Kirill"]
+license = "Unlicense"
+repository = "https://github.com/qoollo/bob-management"
+publish = false
+keywords = [ "BOB", "Management", "GUI" ]
+readme = "./README.md"
+
+[[bin]]
+name = "fullstack"
+path = ".cargo/bin/fullstack.rs"
+required-features = ["dev"]
+
+[[bin]]
+name = "tsync"
+path = ".cargo/bin/tsync.rs"
+
+[[bin]]
+name = "backend"
+path = ".cargo/bin/backend.rs"
+
+[[bin]]
+name = "frontend"
+path = ".cargo/bin/frontend.rs"
+
+[[bin]]
+name = "bob_management"
+path = "backend/src/main.rs"
+
+[dependencies]
+# Axum related
+axum = "0.6"
```

Comment:
Most of the dependencies declared here are for the backend only. Therefore they should be declared only in backend crate and should be removed here